From 9f1f9df48ef8c6f62349eccb90dc54a3c3d75e3f Mon Sep 17 00:00:00 2001
From: Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>
Date: Wed, 31 Jan 2024 12:52:45 +0100
Subject: [PATCH] Remove unused sibling argument from augeas_rm

This makes the function match the method definition. There was already a
warning about this, but Fedora 40 applies more hardening and it's a
fatal error.

Fixes: e6b57c58d24b ("Initial ruby bindings, written by Bryan Kearney")
---
 ext/augeas/_augeas.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/augeas/_augeas.c b/ext/augeas/_augeas.c
index f9b49d1..7ef0d7d 100644
--- a/ext/augeas/_augeas.c
+++ b/ext/augeas/_augeas.c
@@ -184,7 +184,7 @@ VALUE augeas_mv(VALUE s, VALUE src, VALUE dst) {
  *
  * Remove path and all its children. Returns the number of entries removed
  */
-VALUE augeas_rm(VALUE s, VALUE path, VALUE sibling) {
+VALUE augeas_rm(VALUE s, VALUE path) {
     augeas *aug = aug_handle(s);
     const char *cpath = StringValueCStr(path) ;
 
